pallets/drand/substrate-node-template/runtime/src/lib.rs

```diff
@@ -290,7 +290,7 @@ impl pallet_drand::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = pallet_drand::weights::SubstrateWeight<Runtime>;
 	type AuthorityId = pallet_drand::crypto::TestAuthId;
-	type Verifier = pallet_drand::QuicknetVerifier;
+	type Verifier = crate::pallet_drand::verifier::MainnetVerifier;
```

Comment:
If we add a "NetworkType" config value, we could probably remove this or make it an Option. So by default if you use `NetworkType::Quicknet` the verifier is `QuicknetVerifier` but the implementor could add their own verifier if they want to